corehq/warehouse/management/commands/clear_staging_records.py

```diff
@@ -0,0 +1,21 @@
+from django.core.management import BaseCommand
+from corehq.warehouse.const import STAGING_TABLES
+from corehq.warehouse.models import get_cls_by_slug
+
+
+USAGE = """Usage: ./manage.py clear_staging_records
+
+Removes all records from staging tables
```

Comment:
this is actually just for convenience purposes. i have to do this a lot, so felt like just adding a mgmt command to do it